.gitlab-ci.yml

```diff
@@ -711,4 +711,14 @@ trigger-simnet:
     DWNSTRM_ID:                    332
   script:
     # API trigger for a simnet job
-    - .maintain/gitlab/trigger_pipeline.sh --simnet-version=${SIMNET_REF}
+    # use the -v operator to explicitly test if a variable is set
+    # use the -n operator to test if variable is empty string
+    - if [[ -v FEATURES && -n "${FEATURES}" ]]  ; then 
```

Comment:
It's better to just have a conditional flag in the script than writing additional scripts around it